src/app/loadout-builder/filter/FilterBuilds.m.scss

```diff
@@ -44,16 +44,47 @@
   }
 }
 
-.assumeMasterwork {
-  margin-bottom: 4px;
+.upgradeMaterials {
+  padding: 4px;
+  margin: 4px 0;
+  background-color: black;
+}
+
+.materialSelect {
+  width: 100%;
 
-  input {
-    margin-left: 0;
-    vertical-align: middle;
+  button {
+    width: 100%;
   }
+}
 
-  label {
-    cursor: pointer;
-    vertical-align: middle;
+.materialSpendInfo {
+  width: 220px;
+  @include phone-portrait {
+    // width of phone minus 12 padding each side
+    width: calc(100vw - 24px);
   }
 }
+
+.upgradeOption {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  overflow: hidden;
```

Comment:
I am not sure why but for text ellipsis to work correctly I need overflow hidden here and in the `.materialName` class.